promptsource/seqio_tasks/load_annotated_prompts.py

```diff
@@ -0,0 +1,109 @@
+import csv
```

Comment:
Sort of a nitpick (so feel free to disregard) but I would include these functions in tasks.py and then just treat this as a diagnostic script.

Reply:
Sounds good. I will move them in the next commit.